bin/republish_manuals

```diff
@@ -9,13 +9,22 @@ logger.formatter = Logger::Formatter.new
 
 repository = ManualsPublisherWiring.get(:repository_registry).manual_repository
 
-count = repository.all.count
+begin
+  count = repository.all.count
+rescue DocumentAssociationMarshaller::RemovedDocumentIdNotFoundError
+  count = "?"
+end
 
 logger.info "Republishing #{count} manuals..."
 
 repository.all.each.with_index do |manual, i|
-  logger.info("[% 3d/% 3d] id=%s slug=%s" % [i + 1, count, manual.id, manual.slug])
-  ManualServiceRegistry.new.republish(manual.id).call
+  begin
+    logger.info("[ #{i} / #{count} ] id=#{manual.id} slug=#{manual.slug}]")
+    ManualServiceRegistry.new.republish(manual.id).call
+  rescue DocumentAssociationMarshaller::RemovedDocumentIdNotFoundError => e
+    logger.warn("Manual with id=#{manual.id} slug=#{manual.slug} has removed document which was not found: #{e.message}")
```

Comment:
Thanks. Updated message and used 'error' log level in https://github.com/alphagov/manuals-publisher/blob/44d961a9ec26f00d137ce86e5f14d1c4ae7601fd/bin/republish_manuals#L25